fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/BaseTableLoader.java

```diff
@@ -160,7 +161,11 @@ public void loadLogicalTableWithGranularities(
         // For every legal grain
         for (Granularity grain : validGrains) {
             // Build the logical table
-            LogicalTable logicalTable = new LogicalTable(logicalTableName, grain, nameGroup, metricDictionary);
+            LogicalTable logicalTable = new LogicalTable(
+                    (DefaultLogicalTableInfo) () -> logicalTableName,
```

Comment:
Does this work?  This doesn't look like it works.  The fact that this doesn't break a test worries me.

Ok, this almost certainly isn't doing what you think it's doing.